doc/nrf/libraries/others/app_event_manager_profiler_tracer.rst

```diff
@@ -110,22 +110,17 @@ Initialization hook usage
    :start-after: em_initialization_hook_start
    :end-before: em_initialization_hook_end
 
-The Application Event Manager profiled tracer uses the hook to append itself to the initialization procedure.
+The Application Event Manager profiler tracer uses the hook to append itself to the initialization procedure.
+The profiler tracer initializes the :ref:`nrf_profiler` library (:c:func:`nrf_profiler_init`) and registers the nRF Profiler events used to indicate Application Event Manager event submissions, processing starts, and ends.
 
 Tracing hook usage
 ==================
 
-.. include:: app_event_manager.rst
-   :start-after: em_initialization_hook_start
-   :end-before: em_initialization_hook_end
-
-The Application Event Manager profiled tracer uses the hook to append itself to the initialization procedure.
-
 .. include:: app_event_manager.rst
    :start-after: em_tracing_hooks_start
    :end-before: em_tracing_hooks_end
 
-The Application Event Manager profiler tracer uses the tracing hooks to register nRF Profiler events and log their occurrence when application is running.
+The Application Event Manager profiler tracer uses the tracing hooks to log Application Event Manager event submission, processing start, and end as :ref:`nrf_profiler` events while the application is running.
```

Comment:
```suggestion
The Application Event Manager profiler tracer uses the tracing hooks to log the following events while the application is running:

 * Application Event Manager event submission
 * Processing start
 * Processing end as :ref:`nrf_profiler`
```
For better reading. Hope the meaning has not changed.

Reply:
Meaning was changed a bit (not 100% correct). Rephrased - please take a second look